packages/data-point/lib/entity-types/base-entity/resolve.js

```diff
@@ -193,7 +181,7 @@ module.exports.typeCheck = typeCheck
  * @param {Accumulator} accumulator
  * @param {Function} reducer
  * @param {Object} entity
- * @returns {Promise<Accumulator>}
+ * @returns {Promise}
  */
 function resolveEntity (manager, resolveReducer, accumulator, reducer, entity) {
```

Comment:
I also updated the code here to merge the latest code with the intention of the original PR